src/Swarm/TUI/Launch/Controller.hs

```diff
@@ -136,7 +137,24 @@ handleLaunchOptionsEvent siPair = \case
         activateFocusedControl x
       _ -> return ()
   CharKey ' ' -> activateControl
-  Key V.KEnter -> activateControl
+  Key V.KEnter -> do
+    activateControl
+    fr <- use $ uiState . uiLaunchConfig . controls . scenarioConfigFocusRing
+    case focusGetCurrent fr of
+      Just (ScenarioConfigControl (ScenarioConfigPanelControl AutoPlaySelector)) -> do
+        mScenario <- use $ uiState . scenarioRef
```

Comment:
It seems this is not populated yet - where might I find the current scenario when I am in Menu?